### PR TITLE
#25715 orgu - OrgUnitGUI must be called from PluginDispatchGUI to display or…

### PR DESCRIPTION
**Pull request by shecken**
…gu tree in plugin objects

### CODE REVIEW

File: setup/sql/dbupdate_04.php [hunk 1]
```diff
@@ -21741,3 +21741,7 @@ function writeCtrlClassEntry(ilPluginSlot $slot, array $plugin_data) {
 }
 
 ?>
+<#5251>
```
Comment (mjansenDatabay) [inline setup/sql/dbupdate_04.php]: IMO you should use the `5_3_hotfixes.php` file for this update step.

Comment (shecken) [reply setup/sql/dbupdate_04.php]: Thx for hint. I've changed it.

**Comment by kergomard** [comment]
@mstuder : We stumbled upon this PR in our biweekly TB meeting. Could you please merge this PR or provide your objections?
Thank you in advance!

**Comment by mstuder** [comment]
Thx shecken. Sorry about my late reaction :-((!

I fixed this in 5.4 and further:
https://github.com/ILIAS-eLearning/ILIAS/commit/33fafa5df1b0cdfd251beffee4b06a7c1cc93400

**Pull request closed without merging**